### PR TITLE
Remove rewardbook

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

File: modules/fee-market/src/lib.rs [hunk 1]
```diff
@@ -117,8 +116,16 @@ pub mod pallet {
 			Vec<T::AccountId>,
 			Option<T::BlockNumber>,
 		),
-		/// Reward distribute of the order. \[lane_id, message_nonce, rewards\]
-		OrderReward(LaneId, MessageNonce, RewardItem<T::AccountId, BalanceOf<T, I>>),
+		/// Reward distribute of the order. \[lane_id, message_nonce, to_treasury, to_slot_relayer,
+		/// to_message_relayer, to_confirm_relayer\]
+		OrderReward(
```
Comment (HackFisher) [inline modules/fee-market/src/lib.rs:121]: @JayJay1024 Are you good with this event changes?

**Review by HackFisher — COMMENTED** [review]
@AsceticBear please help review this.

**Comment by AsceticBear** [comment]
I’ll deal with this pr after #169 merged since there is some conflict.

File: modules/fee-market/src/lib.rs [hunk 2]
```diff
@@ -117,8 +116,16 @@ pub mod pallet {
...
+			LaneId,
+			MessageNonce,
+			Option<BalanceOf<T, I>>,
+			Option<(T::AccountId, BalanceOf<T, I>)>,
+			(T::AccountId, BalanceOf<T, I>),
+			BalanceOf<T, I>,
```
Comment (JayJay1024) [inline modules/fee-market/src/lib.rs:127]: Here it seems that only amount but no account information for `to_confirm_relayer` ?

**Comment by AsceticBear** [comment]
I have done this in the #169 , Close this one

**Pull request closed without merging**